src/Entities/Blog.php

```diff
@@ -25,6 +23,7 @@ public function __construct(int $id, string $title, string $content, ?int $autho
         $this->postTime = gmdate("d/m/y", strtotime($postTime));
         $this->likes = $likes ?: 0;
         $this->dislikes = $dislikes ?: 0;
+        ($this->likes*1.5)<$this->dislikes ? $this->controversial=true : $this->controversial=false;
```

Comment:
Can you add a test to your Blog tests to make sure this works properly

Reply:
While this ternary works just fine, it's kinda backwards. Because ternaries return a value, you should usually set a variable/property by doing `$this->thing = condition ? true : false;

The way to do it is like:

`$this->controversial = ($this->likes*1.5)<$this->dislikes ? true : false;`

Reply:
Updated ternary and written tests - also updated other tests that use blog entity to include like/dislike params